.github/workflows/workspace_integration_test.yaml

```diff
@@ -1,12 +1,79 @@
-name: Reusable workflow for building Docker images
+name: Workspace Integration Test
 
 on:
   workflow_call:
+    inputs:
+      image_tag:
+        type: string
+    secrets:
+      studio_license_key:
+        required: false
 
 jobs:
-  job_name:
-    name: Any job name with
+  setup:
     runs-on: ubuntu-latest
+    outputs:
+      #We return the image_tag input of the workflow_call or the dotenv action if it is not provided
+      image_tag: ${{ github.event.inputs.image_tag || steps.dotenv.outputs.studio_docker_tag }}
     steps:
-      - name: Hello World
-        run: echo "Hello World after"
\ No newline at end of file
+      - uses: actions/checkout@v4
+      - id: dotenv
+        uses: falti/dotenv-action@v1.0.4
+  integration-test-in-studio-container:
+    needs: setup
+    runs-on: ubuntu-latest
+    env:
+      STUDIO_LICENSE_KEY: ${{ secrets.studio_license_key || secrets.STUDIO_CI_LICENSE_KEY }}
+      # TODO this variable is required but github workspace is different
+      # see https://github.com/actions/runner/issues/2058
+      USER_WS: /github/workspace
```

Comment:
Not sure I understand this part.

Reply:
Me neither :rofl: I mean, this comes from @moriarty code in https://github.com/PickNikRobotics/moveit_studio_ur_ws and I didn't check.
Now, looking into the link I see that there's a confirmed bug related to the paths in the containers in the runners.
I think we could bring some more information about the bug in this comment but keep the hardcoded path.

Reply:
572d25f9e8c5bfa225ed5f644f87bbf737d32dbf